src/main/java/io/debezium/connector/spanner/config/validation/ConnectionValidator.java

```diff
@@ -22,7 +22,7 @@ public class ConnectionValidator implements ConfigurationValidator.Validator {
 
     private static final String GOOGLE_APPLICATION_CREDENTIALS_ENV_VAR = "GOOGLE_APPLICATION_CREDENTIALS";
```

Comment:
Would the environment variable here be enough to detect application deffault credentials?

Reply:
Depending on the authentication method, this environment variable may not be set.
https://cloud.google.com/docs/authentication#auth-decision-tree